content/applications/hr/payroll/payslips.rst

```diff
@@ -221,166 +306,163 @@ To view payslips in batches, navigate to :menuselection:`Payroll app --> Payslip
 
-Use the breadcrumb menu to navigate back to the individual batch detail page, or back to the list of
-all batches.
+- :guilabel:`New`: batches without any payslips added to them have a status of :guilabel:`New`. The
+  following button options appear for these batches:
 
-.. image:: payslips/payslip-batches.png
-   :align: center
-   :alt: Click the Payslips smart button to view the individual payslips in the batch.
+  - :guilabel:`Add Payslips`: click the :guilabel:`Add Payslips` button to add payslips to the
+    batch, and an :guilabel:`Add Payslips` pop-up window appears. Only payslips that can be added to
+    the batch (payslips not currently part of a batch) appear on the list.
 
-Create a new batch
-------------------
+    Select the desired payslips by clicking the checkbox to the left of each payslip name, then
+    click the :guilabel:`Select` button to add them to the batch. Once payslips are selected and
+    added to the batch, the status changes to :guilabel:`Confirmed`.
 
-To create a new batch of payslips from the :guilabel:`Payslips Batches` page
-(:menuselection:`Payroll app --> Payslips --> Batches`), click the :guilabel:`Create` button in the
-upper-left corner. Doing so reveals a blank payslip batch form on a separate page.
+   - :guilabel:`Generate Payslips`: after payslips have been added to the batch, click the
+     :guilabel:`Generate Payslips` button to process the payslips and create individual payslips in
+     the database.
 
-On the new payslip batch form, enter the :guilabel:`Name` for the batch, and select the date range
-to which the batch applies, by clicking the :guilabel:`▼ (drop-down arrow)` icon in the
-:guilabel:`Period` fields, which reveals a calendar pop-up window. From this calendar pop-up window,
-navigate to the correct month, and click on the corresponding day for both the start and end dates.
+     A :guilabel:`Generate Payslips` pop-up window appears. If only a specific :guilabel:`Salary
+     Structure` and/or specific :guilabel:`Department` is desired to make payslips for, select them
+     from the corresponding drop-down menus. If no selections are made, then all payslips listed in
+     the pop-up window are processed as usual.
 
-.. image:: payslips/new-batch-details.png
-   :align: center
-   :alt: Enter the details for the new batch.
+     Click the :guilabel:`Generate` button to create the payslips. The :guilabel:`Generate Payslips`
+     button changes to a :guilabel:`Create Draft Entry` button, and the status changes to
+     :guilabel:`Confirmed`.
 
-If the batch is a credit note, check the box next to :guilabel:`Credit Note`. Then, in the
-:guilabel:`Generation Date` field, select the date the payslips should be generated from a calendar
-pop-up window. This generated date is reflected in the accounting journal entries.
+  .. image:: payslips/batch-new.png
+     :align: center
+     :alt: A batch with a status of new, with the available buttons highlighted.
```

Comment:
I believe this, and the other status-related images in this section of the doc would be more beneficial to reader if placed *between* the status definition and the subsequent buttons/options that appear for that specific status.

So, for this it would be 

New: ,,,,,

image

- Add Payslips: ....

and so on.

Please apply this suggestion for the remainder of screenshots used in this section. Thanks! 👍